vault/dendron.topic.schema.cook.first-schema.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 5U4eAiqshI67VxIL40KWH
+title: Making Your First Schema
+desc: 'Learn to harness the powers of schema and automatically insert templates into new notes'
+updated: 1640789533041
+created: 1640784965050
+---
+
+## Summary
+
+[[Schemas|dendron://dendron.dendron-site/dendron.topic.schema]] make management of new notes easier by automatically inserting Markdown [[templates|dendron://dendron.dendron-site/dendron.topic.templates]] into new notes. This guide will walkthrough making a template for daily journaling, using [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) method, and using schema to handle where the template will be used in your [[hierarchy|dendron://dendron.dendron-site/dendron.topic.hierarchies]].
```

Comment:
- no need to capitalize markdown
- schemas can do more than insert markdown templates, can we make that distinction? eg. `Schemas help you apply consistent structure to all your notes. One of its primary capabilities is to...`

Reply:
> This guide will walkthrough making a template for daily journaling, using [The Five Minute Journal](https://www.intelligentchange.com/products/the-five-minute-journal) method, and using schema to handle where the template will be used in your [[hierarchy|dendron://dendron.dendron-site/dendron.topic.hierarchies]].

This sentence is a little confusing (new user, not sure what it means for `template will be used in your hierarchy`). Change to `and automatically applying that template whenever you create a new journal`

Reply:
this was marked as resolved but its not been updated in the latest PR

Reply:
> and automatically applying that template whenever you create a new journal 

Please also do a sanity check before applying comments. the sentence is no longer grammatical

`(5MJ) method, as an example, and automatically applying that template whenever you create a new journal.`

Reply:
> > schemas can do more than insert markdown templates, can we make that distinction? eg. Schemas help you apply consistent structure to all your notes. One of its primary capabilities is to...
> 
> I see in the new review we completely removed the bit about the schema. Lets not do that but actually make the distinction

This should now appear within the `## Concepts` section, for `### Schemas`, as I updated/modified the [[schema summary|dendron.topic.schema#summary]] I'm showing via note ref:

> **Schema** help you apply consistent structure to all your notes.
>
> As you end up creating more notes, it can be hard to manage them at scale. Think of schemas as an **optional type system** for your notes that describe the hierarchy of your data and are represented as a hierarchy, themselves.
>
> One of the primary capabilities for schema is to automatically insert [[templates|dendron://dendron.dendron-site/dendron.topic.templates]] into new notes.

Reply:
got it. all good then. it was more of an issue since we removed "schema" from the "Summary" section. since the latest adds it back in and the "Concept" section expands on it, we're good